app/src/main/java/org/desarrolladorslp/teckersapp/MainActivity.kt

```diff
@@ -75,8 +75,8 @@ class MainActivity : AppCompatActivity(), View.OnClickListener {
     private fun updateUI(user: FirebaseUser?) {
```

Comment:
There are some places where this method is being call when an error occurs.. when this happens the error should be notified somehow.. (Logging into logcat, show a message to the user, etc..)

Reply:
on which ones??